docs/pages/database-access/guides/redis-cluster.mdx

```diff
@@ -23,18 +23,74 @@ This guide will help you to:
 
 ## Prerequisites
 
-- Teleport version `9.0` or newer.
 - Redis version `6.0` or newer.
 - `redis-cli` installed and added to your system's `PATH` environment variable.
+- A host where you will run the Teleport Database Service. Teleport must be installed.
```

Comment:
Nit: I think that having version (`9.0 or newer`) was useful here since it indicates which release Redis support was added in.